pkg/kubelet/prober/worker.go

```diff
@@ -130,7 +130,10 @@ func (w *worker) run() {
 
 	// If kubelet restarted the probes could be started in rapid succession.
 	// Let the worker wait for a random portion of tickerPeriod before probing.
-	time.Sleep(time.Duration(rand.Float64() * float64(probeTickerPeriod)))
+	jitter := time.Duration(rand.Float64() * float64(probeTickerPeriod))
+	if jitter > time.Since(w.probeManager.start) {
```

Comment:
We only sleep if the probeManager (-> kubelet) has started less than the jitter time ago.